metric-schema-markdown/src/main/java/com/palantir/metric/schema/markdown/MarkdownRenderer.java

```diff
@@ -119,6 +119,13 @@ private static void renderLine(
                     output.append(": ").append(docs);
                 });
                 output.append("\n");
+
+                tagDefinition.getValues().forEach(value -> {
+                    value.getDocs()
+                            .ifPresent(_doc -> output.append(String.format(
+                                    "    - `%s`: %s\n",
+                                    value.getValue(), value.getDocs().get())));
+                });
```

Comment:
I think it's a bit odd that we describe the list of tag values without documentation, then list them again with documentation (example from the test below):

> - `result` values (`failure`,`success`)
>  - `success`: This is a success enum

perhaps when any enum value has documentation, we do away with the `values (each, tag, value)` component and list them all in the new list form as either "- `tagValue`: with documentation" or "- `tagValue`" (without documentation) based on the definition.

I don't have a strong opinion whether we should list all tag values in the new list form, vs only doing so when at least one includes documentation. What do you think?

Reply:
I'm in favour of rendering per line only when any enum value has documentation to keep it more concise.

Reply:
Sounds good. That has the additional upside of not producing a diff in markdown when this is released, and only changing the format when humans add documentation :-)